docs/Inputs.md

```diff
@@ -152,6 +160,20 @@ import { AutocompleteInput, ReferenceInput } from 'react-admin'
 
 **Tip**: React-admin's `<AutocompleteInput>` has only a capital A, while material-ui's `<AutoComplete>` has a capital A and a capital C. Don't mix up the components!
 
+### Properties
+Prop | Required/Optional | Type | Default | Description
+---|---|---|---|---
+`choices` | Required | `Object[]` | - | List of items to autosuggest
+`resource` | Required | `string` | - | The resource working on. This field is passed down by wrapped components like `Create` and `Edit`.  
+`source` | Required |  `string` | - | Name of field to edit, it's type should correspond to the type retrieved from `optionValue` 
+`allowEmpty` | Optional | `boolean` | `false` | If `false` and the searchText typed did not match any suggestion, the searchText will revert to the current value when the field is blurred. If `true` and the `searchText` is set to `''` then the field will set the input value to `null`.
+`inputValueMatcher` | Optional | `Function` | `(input, suggestion, getOptionText) => input.toLowerCase().trim() === getOptionText(suggestion).toLowerCase().trim()` | Allows to define how choices are matched with the searchText while typing.   
+`optionValue` | Optional | `string` | `id` | Fieldname of record containing the value to use as input value 
+`optionText` | Optional | <code>string &#124; Function</code> | `name` | Fieldname of record to display in the suggestion item or function which accepts the currect record as argument (`(record)=> {string}`)
+`setFilter` | Optional | `Function` | null | A callback to inform the `searchText` has changed and new `choices` can be retrieved based on this `searchText`. Signature `searchText => void`. This function is automatically setup when using `ReferenceInput`. 
+`selectedItem` | Optional | `Object` | null | The current selectedItem. If this value is not supplied, it will be reduced from the `choices` list.
```

Comment:
> If this value is not supplied, it will be reduced from the `choices` list.

I don't get this

Reply:
The component expects a selectedItem object to render the `optionText` from. If the selectedItem is not supplied it will look it from the `choices` array (matching the `optionValue` of each record with the `input.value`). The latter was the default behaviour on the previous `AutocompleteInput`. Now it simply allows you supply the `selectedItem` directly (which is what the updated `ReferenceInput` does now).

Reply:
I suggest you put this explanation in the documentation :)

Reply:
Can you remove this prop ?

Reply:
done